src/components/pickup/Pickup.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from "react";
```

Comment:
This file is unnecessary. Just call `PickupTable` in the parent. This is a useless component.

Reply:
Will be resolved in another comment for the PickedUp tab.